### DISCUSSION TITLE
Replace Gitter with Discord server

### DISCUSSION THREAD

**Discussion by MatejNedic** [awspring/spring-cloud-aws#21] [Ideas]
Discussions are good for forum based communication, but for more interactive and faster replies within community I think Discord server would be perfect. 
If there is interest for Discord (in a good way) as another way of communication it should be created and link should be provided.

**Comment by kschlesselmann** [comment]
Why should we switch to discord if all other SC channels remain on gitter?

**Reply by MatejNedic** [reply]
From my experience more people use Discord and have seen in few open source projects questions get answered more quicker. Not only that we can make on one server channel for each integration so your question doesn't get forgotten if few people answer after you which I have seen on gitter

**Comment by eddumelendez** [comment]
One drawback IMO is that there is no threads, so conversations are mixed. I prefer to use github discussions so we can create threads not only for the main discussion but also for every reply.

**Reply by MatejNedic** [reply]
Tbh the more I use discussions more I like them. I thought maybe because Gitter was real-time that we should replace it for real-time service but now when I used Discussions for some time I think you are right, they are a better solution!
